### PR TITLE
Clarify use cases for Angular, React, Vue in README

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: 265215

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/okta-react/README.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+
+- Login and logout from Okta using the [OAuth 2.0 API](https://developer.okta.com/docs/api/resources/oidc)
+- Retrieve user information
+- Determine authentication status
+- Validate the current user's session
+
+All of these features are supported by this SDK. Additionally, using this SDK, you can:
+
+- Add "secure" routes, which will require authentication before render
+- Enter a custom login flow when authentication is required
```
Comment (vijetmahabaleshwar-okta) [inline packages/okta-react/README.md]: I feel "Enter a custom login flow" can be confusing to some users. (Considering that we have IdX pipeline changes) We can probably say - Provide customizable self-hosted login page for authentication or something like that.

Comment (aarongranick-okta) [reply packages/okta-react/README.md]: This is referring to our `onAuthenticationRequired` callback function. They can do almost anything with this function, but our samples typically redirects to a specific `login` route.

Comment (robertjd) [reply packages/okta-react/README.md]: This does read really generic, might be better to be direct, e.g. "Define custom logic/behavior when authentication is required"

File: packages/okta-react/README.md [hunk 2]
```diff
@@ -21,7 +41,7 @@ This library currently supports:
 - [React Quickstart](https://facebook.github.io/react/docs/installation.html#creating-a-new-application)
   - If you don't have a React app, or are new to React, please start with this guide. It will walk you through the creation of a React app, creating routes, and other application development essentials.
 - [Okta Sample Application](https://github.com/okta/samples-js-react)
-  - A fully functional sample application.
+  - A fully functional sample application built using this SDK.
 - [Okta React Quickstart](https://developer.okta.com/quickstart/#/react/nodejs/express)
```
Comment (robertjd) [inline packages/okta-react/README.md]: Let's link to the new guide instead: https://developer.okta.com/docs/guides/sign-into-spa/react/before-you-begin/

File: packages/okta-angular/README.md [hunk 3]
```diff
@@ -1,19 +1,39 @@
...
 This library currently supports:
 
 - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
 - [OAuth 2.0 Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1)
 - [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636)
 
-This library has been tested for compatibility with the following Angular versions: 4, 5, 6, 7, 8
-
-`okta-angular` works directly with [`@angular/router`](https://angular.io/guide/router).
+> This library has been tested for compatibility with the following Angular versions: 4, 5, 6, 7, 8
```
Comment (swiftone) [inline packages/okta-angular/README.md]: Where do we stand on Angular 9?

File: packages/okta-angular/README.md [hunk 4]
```diff
@@ -161,7 +183,7 @@ const appRoutes: Routes = [
 
 ### `OktaLoginRedirectComponent`
 
-By default, the `OktaLoginRedirect` component redirects users to your Okta organization for login. Simply import and add it to your `appRoutes` to offset authentication to Okta entirely:
+The `OktaLoginRedirect` component redirects the user's browser to the Okta-hosted login page for your organization. For more advanced cases, this component can be copied to your own source tree and modified as needed.
```
Comment (swiftone) [inline packages/okta-angular/README.md]: Do we have any examples for this "more advanced case"?  If not that shouldn't hold up this change, but sounds like a good idea of a guide/etc to add and link to from here.

File: packages/okta-angular/README.md (hunk 3 above)
Comment (mraible) [reply packages/okta-angular/README.md]: v1.2.2 of our Angular SDK works with Angular 9. v1.3.0 does not. See https://github.com/okta/okta-oidc-js/issues/643 for more info.

File: packages/okta-angular/README.md (hunk 4 above)
Comment (aarongranick-okta) [reply packages/okta-angular/README.md]: They may want to add a delay w/ an animation, or perhaps do some additional checks before login. Maybe drop a cookie of their own before the redirect for extra validation on the callback. The main idea I want to communicate is they are not forced to use this component. What it does is very simple, but they can copy it if they want something more complex.

**Pull request merged**